packages/apps/plugins/plugin-theme/src/styles.ts

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright 2023 DXOS.org
+//
+
+/**
+ * Cross-cutting styles used by all plugins.
+ */
+// TODO(burdon): Dynamically provided by theme?
+export const Styles = {
+  level0: {
+    bg: 'bg-white dark:bg-neutral-900',
+    text: 'text-black dark:text-neutral-200',
+    border: 'border border-neutral-300 dark:border-neutral-800',
+  },
+  level1: {
+    bg: 'bg-neutral-50 dark:bg-neutral-800',
+    text: 'text-black dark:text-neutral-200',
+    border: 'border border-neutral-300 dark:border-neutral-800',
+  },
+  level2: {
+    bg: 'bg-neutral-100 dark:bg-neutral-700',
+    text: 'text-black dark:text-neutral-200',
+    border: 'border border-neutral-300 dark:border-neutral-800',
+  },
+};
```

Comment:
Please check with someone familiar with `aurora-theme` before proliferating semantic design system tokens like this — we already have something similar in `aurora-theme`’s `fragments`, which are exported and can be used.